src/docs/design/design-audit.mdx

```diff
@@ -0,0 +1,212 @@
+import { ImageGrid, RuleTable, TOC } from '../../app/components';
+import Banner from '../../app/assets/design/design-audit/banner.svg';
+import TopSeverity from '../../app/assets/design/design-audit/top-severity.png';
+import HighSeverity from '../../app/assets/design/design-audit/high-severity.png';
+import MediumSeverity from '../../app/assets/design/design-audit/medium-severity.png';
+import LowSeverity from '../../app/assets/design/design-audit/low-severity.png';
+import Feature from '../../app/assets/design/design-audit/feature.png';
+import Slides from '../../app/assets/design/design-audit/ppt.svg';
+import DoDont from '../../app/assets/design/design-audit/do-dont.svg';
+import Educational1 from '../../app/assets/design/design-audit/educational-1.png';
+import Educational2 from '../../app/assets/design/design-audit/educational-2.png';
+
+<!-- audit critique review -->
+
+# UI Design Audits
+
+A design audit is a great way to ensure that your user interface design meets Brightlayer UI’s design standard and lines up with other Brightlayer UI-based applications. This process examines not only branding elements, but also the usability of products.
+
+<ImageGrid
+    images={[Banner]}
+    regularWidth
+    caption={'A typical UI audit will highlight problems and provide design recommendations.'}
+/>
+
+Do not wait until a product is about to be released to request for design audits. Design audits are not just about the cosmetic alignments, but focused more on the usability side, especially when the product team does not have a professional UI/UX designer. Design audits themselves can take weeks, depending on the scope, and it can take months to implement the top- / high-severity issues identified.
```

Comment:
```suggestion
Don't wait until a product is ready for release to request a design audit. Design audits are not just about the cosmetic alignments, but also focus on the overall usability side — this is especially important when your team does not have a professional UI/UX designer on staff. The design audit itself may take several weeks, depending on the scope, and implementing the recommended (high-severity) changes could take months.
```

Reply:
Don't wait until a product is ready for release to request a design audit. Design audits are not just about the cosmetic alignments, but also focus on the overall usability side — this is especially important when your team does not have a professional UI/UX designer on staff. The design audit itself may take several weeks, depending on the scope, and implementing the recommended changes could take months.

Reply:
```suggestion
Don't wait until a product is ready for release to request a design audit. Design audits are not just about the cosmetic alignments, but also focus on the overall usability side — this is especially important when your team does not have a professional UI/UX designer on staff. The design audit itself may take several weeks, depending on the scope, and implementing the recommended changes could take months.
```